micromasters/settings.py

```diff
@@ -257,7 +257,8 @@ def get_var(name, default):
 
 # Request files from the webpack dev server
 USE_WEBPACK_DEV_SERVER = get_var('MICROMASTERS_USE_WEBPACK_DEV_SERVER', False)
-WEBPACK_SERVER_URL = get_var('MICROMASTERS_WEBPACK_SERVER_URL', 'http://{host}:8078')
+WEBPACK_DEV_SERVER_HOST = get_var('WEBPACK_DEV_SERVER_HOST', '')
```

Comment:
when this isn't specified, we want the webpack server host to be the same as the django host. before this PR, we were pulling the django host value out of the request in our views (`request.get_host()`). i kept that approach. when `WEBPACK_DEV_SERVER_HOST` is blank, we get the host via `request.get_host()`. i mentioned above that if there's an easy way to get the django host that doesn't involve pulling it out of the request, we should go with that